### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by wustzhaohui**
Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,15 +68,15 @@ const App = () => {
 - [x] Search
 - [x] Slider
 - [x] Stepper
-- [ ] Switch
-- [ ] Uploader
+- [x] Switch
+- [x] Uploader
```
Comment (ssysm) [inline README.md:72]: Can you double-check if the uploader is up on the master branch? I don't see it in the storybook.

**Review by ssysm — COMMENTED** [review]
Need more investigation

**Review by ssysm — DISMISSED** [review]
all features presented in storybook.

**Comment by wustzhaohui** [comment]
> all features presented in storybook.

waht do you mean?

**Comment by ssysm** [comment]
All good ready for merge

**Comment by wustzhaohui** [comment]
> All good ready for merge

I think you should check the checkStatus when a new coponents request to merge.

It's troublesome to check if the component finished after the code has merged.

**Pull request merged**